contracts/test/PoolDelegate.t.sol

```diff
@@ -344,12 +382,20 @@ contract PoolTest is TestUtil {
         assertEq(lee_bal_post - lee_bal_pre, principal_lee + interest_lee);
     }
 
-    function test_reclaim_erc20() external {
+    function test_reclaim_erc20(
+        uint256 mintedUsdc,
+        uint256 mintedDai,
+        uint256 mintedWeth
+    ) external {
```

Comment:
We also should just leave this as a unit test IMO

Reply:
Again it is cherry on top, No harm